lib/redis/namespace.rb

```diff
@@ -547,5 +547,21 @@ def create_enumerator(&block)
         Enumerator.new(&block)
       end
     end
+
+    def full_prefix
+      ancestor_namespaces.reverse.join(':')
+    end
+
+    def ancestor_namespaces
+      @ancestor_namespaces ||= begin
+        parent_namespaces =
+          if @redis.respond_to?(:ancestor_namespaces, true)
+            @redis.send(:ancestor_namespaces)
```

Comment:
I think this method should be public but not part of the public API so we can avoid the `send`.

Reply:
I can make this method ```protected```. 
It will give the ability to call the method inside the class body without ```send``` and it will not be visible from outside.
@rafaelfranca It's ok?

Reply:
I just added a similar method, public. We can use it instead.

Reply:
Done.